src/annotator/config/index.js

```diff
@@ -111,11 +151,19 @@ function configFrom(settingsKeys, window_) {
  * @param {'sidebar'|'notebook'|'annotator'|'all'} [appContext] - The name of the app.
  */
 export function getConfig(appContext = 'annotator', window_ = window) {
+  const settings = settingsFrom(window_);
```

Comment:
I left settings here instead of putting it inside of configDefinitions because later on we need to see if  `settings.isBrowserExtension` is true as that logic will move out of settings into this method. However, I may revisit that and see if there is a more straightforward way to get that without using settings.js file.  If that's the case, this var would be better place inside of configDefinitions() rather than being passed in.